build/webpack/_production.js

```diff
@@ -16,17 +16,37 @@ export default (webpackConfig) => {
   }
 
   debug('Apply ExtractTextPlugin to CSS loaders.')
+  let appliedExtractCSS = false
   webpackConfig.module.loaders = webpackConfig.module.loaders.map(loader => {
-    if (/css/.test(loader.test)) {
-      const [first, ...rest] = loader.loaders
+    if (!loader.loaders) {
+      return loader
+    }
+
+    let hasCSSLoader = false
+    loader.loaders.forEach(styleLoader => {
```

Comment:
you could add anchor operators to the regex ie `/^css$/.test(loader.split('?')[0])`

> I only mention the css shorthand since I was actually planning on removing the -loader suffixes.
:+1: the `-loader` is unnecessary